pages/profile/v2/index.tsx

```diff
@@ -32,30 +33,31 @@ export default function Profile(props) {
   return (
     <div className="flex flex-col p-4 m-4 overflow-auto bg-scroll">
       <ProfileHeaderComponent userId={user.uid} />
-
-      <h2 className="text-lg font-bold mt-14 mb-9">Assignments</h2>
-
-      <div className="grid grid-cols-1 mb-4">
-        <AssignmentSectionComponent />
+      <div className="grid grid-cols-1 my-9">
```

Comment:
you can remove this 1 column grid from all these lines, i'd also recommend removing my-9 and using space-y-[] in the parent div